resources/content/schema/admin/admin-auth.json

```diff
@@ -1,46 +1,69 @@
 {
     "authorize" : {
-        "[a-zA-Z]*" : "crud",
-        "[a-zA-Z]*\\..*" : "r",
-
-        ".*\\..*Id" : "cru",
-        ".*\\.data" : "cru",
+    	".*" : "",
+    	
+    	".*\\..*accountId" : "r",
```

Comment:
Should be:
```
".*\\.accountId" : "r",
```
You only want it it apply to <resource>.accountId, nothing else. Having that extra ```.*``` in there would allow it to apply to <resource>.fooBaraccountId, which is not what we want.